src/components/buttons/aprove-server-form.js

```diff
@@ -41,5 +41,15 @@ module.exports = {
         await interaction.message.thread.setArchived(true);
 
         await interaction[interaction.replied || interaction.deferred ? 'followUp' : 'reply']({ content: "Servidor Aprovado", ephemeral: true });
+
+        const webhook = parseWebhookURL(process.env.OFFTOPIC_WEBHOOK);
+        await interaction.client.fetchWebhook(webhook.id, webhook.token)
+            .then((webhook) => {
```

Comment:
mesma coisa do outro arquivo
```suggestion
            .then(async (webhook) => {
```